samples/bluetooth/hci_uart/rv32m1_vega_ri5cy.conf

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2020, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_CONSOLE=n
+CONFIG_STDOUT_CONSOLE=n
+CONFIG_UART_CONSOLE=n
+CONFIG_GPIO=y
+CONFIG_SERIAL=y
+
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_MAX_CONN=16
+CONFIG_BT_TINYCRYPT_ECC=n
+CONFIG_BT_CTLR_DTM_HCI=y
+CONFIG_BT_CTLR_ASSERT_HANDLER=y
+
+#CONFIG_HAS_MCUX_LPUART=y
+#CONFIG_UART_MCUX_LPUART_0=y
+#CONFIG_UART_MCUX_LPUART=y
```

Comment:
remove these

Reply:
We've seemed to miss those; I'm resending ASAP